.github/workflows/test.yml

```diff
@@ -160,64 +160,3 @@ jobs:
       if: ${{ always() }}
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
-
-  docker:
-    # build and test docker image
-    if: ${{ contains(needs.check_skip_flags.outputs.head-commit-message, 'full_test') }}
-    runs-on: ubuntu-latest
-    needs: [download-test-data, check_skip_flags]
-    steps:
-      - uses: actions/checkout@v4
-      - name: Build the Docker image
-        run: |
-          user_name="bids"
-          docker build . --file Dockerfile --tag ${user_name}/giga_connectome
-          mkdir -p "${HOME}/docker"
-          docker save "${user_name}/giga_connectome" > "${HOME}/docker/image.tar"
-
-      - name: Restore cached data ds000017
-        id: cache
-        uses: actions/cache/restore@v4
-        with:
-          path: /home/runner/work/giga_connectome/giga_connectome/giga_connectome/data/test_data
-          key: ds000017
-
-      - name: Test the Docker image
-        run: |
-          docker load -i ${HOME}/docker/image.tar
-          docker run --rm --read-only \
-            -v /home/runner/work/giga_connectome/giga_connectome/giga_connectome/data/test_data:/test_data \
-            -v ./outputs:/outputs \
-            -v ./outputs/working_dir:/work \
-            bids/giga_connectome \
-            /test_data/ds000017-fmriprep22.0.1-downsampled-nosurface \
-            /outputs \
-            participant \
-            -w /work \
-            --participant_label 1 \
-            --reindex-bids
-
-      - name: Upload output artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: connectome
-          path: ./outputs/
-      - name: Upload docker image artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: docker-image
-          path: ${HOME}/docker/image.tar
```

Comment:
probably worth transferring to the other workflow to be able to inspect the output